website/content/api-docs/agent/check.mdx

```diff
@@ -220,6 +220,10 @@ The table below shows this endpoint's support for
   results required before check status transitions to passing. Available for HTTP,
   TCP, gRPC, Docker & Monitor checks.
 
+- `FailuresBeforeWarning` `(int: 0)` - Specifies the number of consecutive unsuccessful
+  results required before check status transitions to warning. Defaults to the same
+  value as `FailuresBeforeCritical`. Available for HTTP, TCP, gRPC, Docker & Monitor checks. 
```

Comment:
Looks good! We should add the "Added in Consul 1.11" here too. We're missing that for the other fields, but we could add that later .

Reply:
Sure.